README.md

```diff
@@ -79,7 +72,9 @@ make (sandbox|preview|staging|production) cf-deploy
 
 ## Running locally
 
-During development it may be preferable to run locally - in which case you'll need to run the following steps
+During development it may be preferable to run locally.
+
+If you haven't created the virtualenv follow these steps - 
```

Comment:
hmm if we're cleaning up the file, then this should probably have two sub-sub-sub headings - `first time installation` and `running the app` or similar, so it's apparent that you only need to install once. then we don't need to say "if you haven't created the virtualenv"